changelog.md

```diff
@@ -9,7 +9,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
-*  Update `queryTip` and `queryTipSlotNo` in `Convex.Devnet.NodeQueries` to also return `SlotLength`.
+* Update `queryTip` and `queryTipSlotNo` in `Convex.Devnet.NodeQueries` to also return `SlotLength`.
+* Add `querySlotNo` to `MonadBlockchain` typeclass and update both blockchain and mockchain implementations.
+* Add `utcTimeToPosixTime` in `Convex.Utils`.
```

Comment:
You can put this in the "Added" section. Generally I'm trying to follow this guide: https://keepachangelog.com/en/1.1.0/